core/src/test/java/io/questdb/griffin/engine/groupby/SampleByTest.java

```diff
@@ -2558,6 +2558,62 @@ public void testSampleByMillisFillNoneNotKeyedEmpty() throws Exception {
                 false);
     }
 
+    @Test
+    public void testSampleByMicrosFillNoneNotKeyedEmpty() throws Exception {
+        assertQuery("sum\tk\n",
+                "select sum(a), k from x sample by 100U fill(none)",
+                "create table x as " +
```

Comment:
This statement does create the table, however the `long_sequence(0)` means that no data are inserted, did you do this in purpose?, what is the reason?, did you mean to simply create the table:

```

create table x (
    a double,
    b symbol,
    k timestamp
) timestamp(k) partition by none
```

and then populate it with the subsequent insert from select statement?.

Reply:
Yeah, you're right! And I'll change my SQL commands.